day01/Husky/src/assets/sound.ts

```diff
@@ -0,0 +1,23 @@
+import boom from "../../../sample/sounds/boom.wav";
```

Comment:
if relative path become verbose, using path alias to improve

Reply:
fixed